src/Orleans.Runtime/Transactions/ITransactionManagerService.cs

```diff
@@ -1,16 +1,8 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Threading.Tasks;
 
 namespace Orleans.Transactions
 {
-    public interface ITransactionManagerService
-    {
-        Task<StartTransactionsResponse> StartTransactions(List<TimeSpan> timeouts);
-        Task<CommitTransactionsResponse> CommitTransactions(List<TransactionInfo> transactions, HashSet<long> queries);
-        Task AbortTransaction(long transactionId, OrleansTransactionAbortedException reason);
-    }
-
     [Serializable]
     public struct CommitResult
```

Comment:
Are the structs in this file still used somewhere? I can't find any occurrences.